mis1946/autoapp
# Bug Fixes: Update Customer Vehicle Info, Vehicle Description Parameter and Added Inquiry Refund

- Replaced button under inquiry process: btnProcess, btnModify and btnApply at main button area.
- Added minor script for customer vehicle info entry.
- Added minor script for Vehicle Ingine Frame Format Entry.
- Call clientvehicleinfo class and test setting of data to class.
- Populate vehicle info table.
- Table: Commented unused arguments.
- Test class with gui: Add, Edit and Save and search for MAKE, MODEL, TYPE, TRANSMISSION, COLOR AND YEAR and Search for available vehicle.
- Added retrieval of currently saved data on btnSave.
- Added main functionality for engine frame entry.
- Added setting of make and model ID and Description when opening engine frame entry window.
- Test visibility of fields and
- Added validation for engine and frame before setting value to class.
- Added autoset of codetype when opening of engine frame entry is triggered at textfield.
- Added opening of new tab for vehicle description entry.
- Change to public AnchorPane AnchorMain; to be able to access on customer vehicle entry.
- Disabled sortable and editable columns on table for vehicle info and vehicle history.
- Disabled fields when user select vehicle thru available vehicle search.
- Uncomment functionality of EDIT for engine and frame entry
- Reset value for engine and frame when user changed make and model
- Reset value of make, model, type, transmission, color and year where one of fields was changed.
- Added functionality for promo, sales call and event.
- Added okaycancel message box on btnCancel.
- Standardize message boxes info.
- Added set to empty string on search of vehicle size, variant a and b when no record found.
- Added Clearfields for new added columns on class: sPlatform and activity title.
- Added oTransFollowUp.LostSale() on tag lost sale button.
- Fixed bug in load fields: replace to getMaster instead of getting values thru table class.
- Removed commented script.
- Added Validation on visibility of buttons.
- Added load for refund print.
- Created new fxml for refund print and added functionality.
- Created new fxml for refund print.
- Added class for populating inquiry refund.
- Created jasper report for inquiry refund print.
- Added parameter for editable refund amount.
- Added text field for refund amount value.
- Mandatory Hide Print and Save button via jasper report and replace it to Print Button at GUI to check when user click print / save.
- Added Print and Cancel Button.
- Added Parameter for Prepared by, Refund No and Date and sourcecode.
- Fixed clearing fields when search at btn available unit return false: Replaced clearVehicleInfoFields(); when search result is true.
- Fixed bugs on Vehicle Parameter
- Fixed clearing fields when search at btn available unit return false: Replaced clearVehicleInfoFields(); when search result is true.
- Created GUI for Vehicle Transfer Ownership.
- Created Class for opening parameter window at menu.
- Call Vehicle Transfer ownership window.
- Added validation on setting of Engine and Frame entry.
- Added documentation for class.
- Added KeyPressed function for Make and Model Search.
- Added KeyPressed function for Make Search.
- Added setOpenEvent for Identifier where the parameter window was opened.
